### PR TITLE
Rename container labels to io.k0sproject.footloose

### PR DESCRIPTION
**Pull request by kke**
Replaces the labels as suggested in https://docs.docker.com/config/labels-custom-metadata/#key-format-recommendations

### CODE REVIEW

**Comment by jnummelin** [comment]
hmm, do we need to worry about backwards compatibility (for our internal purposes even)?

**Comment by kke** [comment]
To me looks like the labels are only used when creating machines and in footloose's own test suite.

File: pkg/cluster/cluster.go [hunk 1]
```diff
@@ -597,16 +597,11 @@ func (f *matchFilter) Write(p []byte) (n int, err error) {
...
+//ssh_exchange_identification: read: Connection reset by peer
 var connectRefused = regexp.MustCompile("^ssh_exchange_identification: ")
 
-// Matches:
-//
-//	
-//	Warning:Permanently added '172.17.0.2' (ECDSA) to the list of known hosts
-var knownHosts = regexp.MustCompile("^Warning: Permanently added .* to the list of known hosts.")
+// Warning:Permanently added '172.17.0.2' (ECDSA) to the list of known hosts
+va  knownHosts = regexp.MustCompile("^Warning: Permanently added .* to the list of known hosts.")
```
Comment (kke) [inline pkg/cluster/cluster.go:604]: what

Comment (kke) [reply pkg/cluster/cluster.go:604]: looks like the ioutil PR broke the file

Comment (kke) [reply pkg/cluster/cluster.go]: yeah, main is broken.

File: pkg/cluster/cluster.go [hunk 2]
```diff
@@ -597,14 +597,10 @@ func (f *matchFilter) Write(p []byte) (n int, err error) {
 }
 
 // Matches:
-//
-//
-//	h_exchange_identification: read: Connection reset by peer
```
Comment (kke) [inline pkg/cluster/cluster.go:602]: I think this was broken in main by the ioutil PR

**Review by twz123 — APPROVED** [review]

**Pull request merged**